jetbrains-core/src/software/aws/toolkits/jetbrains/services/cwc/controller/chat/telemetry/TelemetryHelper.kt

```diff
@@ -94,36 +99,30 @@ class TelemetryHelper(private val context: AmazonQAppInitContext, private val se
             cwsprChatSourceLinkCount = response.relatedSuggestions?.size,
             cwsprChatReferencesCount = 0, // TODO
             cwsprChatFollowUpCount = response.followUps?.size,
-            cwsprChatTimeToFirstChunk = getResponseStreamTimeToFirstChunk(response.tabId),
-            cwsprChatTimeBetweenChunks = "", // getResponseStreamTimeBetweenChunks(response.tabId), //TODO: allow '[', ']' and ',' chars
+            cwsprChatTimeToFirstChunk = getResponseStreamTimeToFirstChunk(response.tabId).toInt(),
+            cwsprChatTimeBetweenChunks = "[${getResponseStreamTimeBetweenChunks(response.tabId).joinToString(",")}]",
```

Comment:
Can you provide evidence for:

1. This change has been tested end to end with service. The API call responded with 200. 
2. The field `cwsprChatTimeBetweenChunks` can be de-serialized and it has the same schema as the one defined in https://github.com/aws/aws-toolkit-vscode/pull/4200/files

Reply:
Here are some request Ids generated from RTS prod:
1. Add message event: `effada91-43f0-4dcb-8098-7ca3e914bf19`
2. Interact with message event: `6b24391c-f526-4db2-8313-fe209e8cb14b`
3. User modification event: `31030da3-b944-4c25-945c-c734064760ed`

Reply:
Shared some request Ids offline.